packages/mobx/src/types/type-utils.ts

```diff
@@ -92,3 +98,22 @@ export function getDebugName(thing: any, property?: string): string {
     }
     return named.name_
 }
+
+/**
+ * Helper function for initializing observable structures, it applies:
+ * 1. allowStateChanges so we don't violate enforceActions.
+ * 2. untracked so we don't accidentaly subscribe to anything observable accessed during init in case the observable is created inside derivation.
+ * 3. batch to avoid state version updates
+ */
+export function initObservable<T>(cb: () => T): T {
```

Comment:
NIT: this feels so core it might fit better into atom.ts or derivation.ts? Not sure

Reply:
It's helper for initializing "complex" observable structures - things inside `types` dir, but I don't have strong opinions about it.

Reply:
Fair enough!